policyengine_canada/variables/gov/income_tax.py

```diff
@@ -10,7 +10,7 @@ class income_tax(Variable):
     definition_period = YEAR
 
     def formula(person, period, parameters):
-        income = person("employment_income", period)
+        income = person("total_individual_pre_tax", period)
```

Comment:
This should fix the [test failure](https://github.com/PolicyEngine/policyengine-canada/actions/runs/3317017617/jobs/5481639569).
```suggestion
        income = person("total_individual_pre_tax_income", period)
```